paddle/gserver/tests/test_LayerGrad.cpp

```diff
@@ -1685,7 +1685,7 @@ TEST(Layer, smooth_l1) {
   config.layerConfig.add_inputs();
 
   for (auto useGpu : {false, true}) {
-    testLayerGrad(config, "smooth_l1", 100, false, useGpu, false, 2.0);
+    testLayerGrad(config, "smooth_l1", 100, false, useGpu, false);
   }
 }
 
```

Comment:
Can you extend the loss test to have more than 1 dim output ? 

```
  config.inputDefs.push_back({INPUT_DATA, "layer_0", 1, 0});
  config.inputDefs.push_back({INPUT_DATA_TARGET, "layer_1", 1, 0});
```

Change the size of input and label to be 10 in order to make sure it works with more than 1 dim